ci/terraform/oidc/mfa-reset-authorize.tf

```diff
@@ -0,0 +1,66 @@
+module "mfa_reset_authorize_role" {
+  source      = "../modules/lambda-role"
+  environment = var.environment
+  role_name   = "mfa-reset-authorize-role"
+  vpc_arn     = local.authentication_vpc_arn
+
+  #TODO: enable when used
```

Comment:
I'd say not the end of the world to enable these now rather than leave a TODO in prod. Alternatively stick a ticket number on the TODO, ie. "will be enabled in ATO-XYZ"

Reply:
Have added the follow up ticket number ATO-670 in the TODO